sql/core/src/main/scala/org/apache/spark/sql/execution/columnar/compression/CompressibleColumnBuilder.scala

```diff
@@ -46,6 +47,7 @@ private[columnar] trait CompressibleColumnBuilder[T <: AtomicType]
   this: NativeColumnBuilder[T] with WithCompressionSchemes =>
 
   var compressionEncoders: Seq[Encoder[T]] = _
+  private val alignedArch = Set("sparc", "sparcv9").contains(System.getProperty("os.arch"))
```

Comment:
Sorry, mis-understanding that you meant to put it into a companion object.  Will get that taken care of.